config/initializers/_service_providers.rb

```diff
@@ -2,3 +2,9 @@
                     fetch(Rails.env, {})
 
 VALID_SERVICE_PROVIDERS = JSON.parse(Figaro.env.valid_service_providers)
```

Comment:
I still think this could be `.to_set` here and simplify `service_provider_updater` but it's not a big deal